src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.Parse.cs

```diff
@@ -12,6 +12,8 @@ namespace System.Text.Json
 {
     public sealed partial class JsonDocument
     {
+        private static readonly byte[] Utf8Bom = { 0xEF, 0xBB, 0xBF };
```

Comment:
```suggestion
        private static ReadOnlySpan<byte> Utf8Bom => new byte[] { 0xEF, 0xBB, 0xBF };
```
Its what all the cool kids are doing these days: https://vcsjones.com/2019/02/01/csharp-readonly-span-bytes-static/

Reply:
I was going to say that my concern with that would be that it might impact the source-package; but it looks like other things in this library already do it: https://github.com/dotnet/corefx/blob/d856beaad11581ff9c6100e73ceb427eb1f1febf/src/System.Text.Json/src/System/Text/Json/JsonConstants.cs#L26-L28

So... reasonable suggestion :smile: